db/migrate/1726547853383_add_uuid_to_books.rb

```diff
@@ -0,0 +1,9 @@
+class AddUuidToBooks < ActiveRecord::Migration[7.0]
+  def up
+    add_column :books, :uuid, :string, limit: 255
```

Comment:
To improve performance, consider adding an index to the `uuid` column. This will speed up queries that filter or join on this column.